libvast/test/pipeline.cpp

```diff
@@ -192,9 +190,8 @@ TEST(head 0) {
 }
 
 TEST(head 10 with overlap) {
-  const auto* vastql
-    = vast::plugins::find<vast::query_language_plugin>("vastql");
-  auto [expr, pipeline] = unbox(vastql->make_query("head"));
+  const auto* vast = vast::plugins::find<vast::language_plugin>("VAST");
```

Comment:
Why is this now capitalized?

Reply:
Because the `VAST` plugin `name()` is also capitalized. Are `plugins::find()` parameters supposed to be not capitalized?

Reply:
So far I thought we always use lower-case names for plugins. But I realize this is independent of this PR, so we'll just keep it as is.